.github/workflows/docker_img.yaml

```diff
@@ -69,5 +69,5 @@ jobs:
               uses: crazy-max/docker-scan-action@master
               with:
                   # NOTE: This task validates the images built previously with latest tag
-                  image: connectedhomeip/chip-build${{ matrix.img }}:latest
+                  image: connectedhomeip/chip-build${{ matrix.img }}:0.5.70
```

Comment:
I think that the version here should be "latest", so the CI will verify lastly build Docker image.